benchmarks/src/garage_benchmarks/experiments/algos/her_garage_tf.py

```diff
@@ -6,7 +6,7 @@
 from garage.envs import GarageEnv, normalize
 from garage.experiment import deterministic
 from garage.np.exploration_policies import AddOrnsteinUhlenbeckNoise
-from garage.replay_buffer import HerReplayBuffer
+from garage.replay_buffer import HERReplayBuffer
```

Comment:
perhaps it should now be HERPathBuffer? No need to block on this, though.

Reply:
I'll make this change along with the replay buffer API.